packages/network/package.json

```
@@ -20,6 +20,7 @@
     "debug": "^4.3.2",
     "fs-extra": "9.1.0",
     "lodash": "^4.17.21",
+    "minimatch": "^3.0.5",
```

Should we keep this `3.0.5` to match the versioning of minimatch used throughout the repo?

---

That was the intent. I kept it at `^3.0.0` for that reason for consistency of the API. We can also upgrade it but might be confusing when comparing against different packages

---

The others are locked down though 😬 
```suggestion
    "minimatch": "3.0.5",
```

---

ill pin it!